game/scripts/npc/abilities/axe_battle_hunger.txt

```diff
@@ -64,7 +64,7 @@
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage_per_second"                               "16 24 32 40 80 120"
+        "damage_per_second"                               "20 28 36 44 76 124" //OAA
```

Comment:
Total damage is still lower than before. I didn't leave dmg the same at early levels because it will be very bad. Mana cost is also to nerf race-car Axe in the first few minutes of the game. I am buffing mid and late-game values because this spell is useless then.